Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,73 @@
+Team 4 – Mario Kart & Ride Replay Simulator
+Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham
+Detail Design: Resistance System
+
+
+Function of the Subsystem: 
+The function of the resistance system is to wirelessly receive an input value from the RRS processing subsystem. The input value is to be received by the H-Bridge controller. The H-Bridge controller will send a power signal to the linear actuator in the form of a pulse width modulated value to adjust the position of the magnets across the flywheel resulting in the creation of eddy currents in the flywheel. These eddy currents create a magnetic field that opposes the source magnetic field. This causes power dissipation in the flywheel, as well as opposing torque, which creates a change of resistance felt by the user. 
+
+Constraints:
+Cost: The main constraint for the resistance system is the cost of materials for both the flywheel, linear motor, and magnets. The total budgeted cost of the resistance system is 50% of the project budget, equaling $600. 
+-	Flywheel: A traditional spin bike flywheel used commercially ranges from $270 to $475 used. To mitigate the cost of the flywheel, team 4 will construct a flywheel using the existing bike tire and aluminum sheet material.
```

Comment:
This seems like a bad idea. Have you consulted any mechanical engineering staff?